From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Tue, 15 Nov 2022 17:24:01 +0800
Subject: disable-crypto

---
 src/os/CMakeLists.txt | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/os/CMakeLists.txt b/src/os/CMakeLists.txt
index 760244b..65c0d4c 100644
--- a/src/os/CMakeLists.txt
+++ b/src/os/CMakeLists.txt
@@ -113,8 +113,9 @@ endif()
 target_link_libraries(os kv)
 
 add_dependencies(os compressor_plugins)
-add_dependencies(os crypto_plugins)
-
+if(HAVE_INTEL AND HAVE_BETTER_YASM_ELF64 AND (NOT APPLE))
+    add_dependencies(os crypto_plugins)
+endif()
 
 if(WITH_BLUESTORE)
   add_executable(ceph-bluestore-tool
