Description: Resolve build failure with boost 1.74 and c++ 17
Author: James Page <james.page@ubuntu.com>
Forwarded: no

--- a/src/librbd/AsioEngine.h
+++ b/src/librbd/AsioEngine.h
@@ -39,7 +39,8 @@ public:
   inline operator boost::asio::io_context&() {
     return m_io_context;
   }
-  inline boost::asio::io_context::executor_type get_executor() {
+  typedef boost::asio::io_context::executor_type executor_type;
+  inline executor_type get_executor() {
     return m_io_context.get_executor();
   }
 
