From: Luciano Lo Giudice <luciano.logiudice@canonical.com>
Date: Fri, 14 Jun 2024 09:45:10 +0800
Subject: Build the QAT libs on x86-64 instead of !aarch64

Forwarded: no
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a8b7c8e..2743822 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -309,11 +309,11 @@ endif()
 option(WITH_BLUEFS "libbluefs library" OFF)
 
 CMAKE_DEPENDENT_OPTION(WITH_QATLIB "Enable QAT with qatlib" ON
-  "NOT CMAKE_SYSTEM_PROCESSOR MATCHES aarch64" OFF)
+  "CMAKE_SYSTEM_PROCESSOR MATCHES i386|i686|amd64|x86_64|AMD64" OFF)
 option(WITH_SYSTEM_QATLIB "Use system packages for qatlib" OFF)
 option(WITH_QATDRV "Enable QAT with out-of-tree driver" OFF)
 CMAKE_DEPENDENT_OPTION(WITH_QATZIP "Enable QATzip" ON
-  "NOT CMAKE_SYSTEM_PROCESSOR MATCHES aarch64" OFF)
+  "CMAKE_SYSTEM_PROCESSOR MATCHES i386|i686|amd64|x86_64|AMD64" OFF)
 option(WITH_SYSTEM_QATZIP "Use system packages for QATzip" OFF)
 
 if(WITH_QATDRV)
