Description: Build the QAT libs on x86-64 instead of !aarch64
Author: Luciano Lo Giudice <luciano.logiudice@canonical.com>
Forwarded: no

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -309,11 +309,11 @@
 option(WITH_BLUEFS "libbluefs library" OFF)
 
 CMAKE_DEPENDENT_OPTION(WITH_QATLIB "Enable QAT with qatlib" ON
-  "NOT CMAKE_SYSTEM_PROCESSOR MATCHES aarch64" OFF)
+  "CMAKE_SYSTEM_PROCESSOR MATCHES i386|i686|amd64|x86_64|AMD64" OFF)
 option(WITH_SYSTEM_QATLIB "Use system packages for qatlib" OFF)
 option(WITH_QATDRV "Enable QAT with out-of-tree driver" OFF)
 CMAKE_DEPENDENT_OPTION(WITH_QATZIP "Enable QATzip" ON
-  "NOT CMAKE_SYSTEM_PROCESSOR MATCHES aarch64" OFF)
+  "CMAKE_SYSTEM_PROCESSOR MATCHES i386|i686|amd64|x86_64|AMD64" OFF)
 option(WITH_SYSTEM_QATZIP "Use system packages for QATzip" OFF)
 
 if(WITH_QATDRV)
